apps/antalmanac/src/stores/calendarizeHelpers.ts

```diff
@@ -144,8 +144,8 @@ export const calendarizeCustomEvents = (currentCustomEvents: RepeatingCustomEven
  * @returns the start and end time of a course in a 24 hour time with a leading zero (##:##)
  * @returns undefined if there is no WebSOC time (e.g. 'TBA', undefined)
  */
-export const translateWebSOCTimeTo24HourTime = (section: AASection) => {
-    const timeString = section.meetings[0].time.replace(/\s/g, '');
+export const translateWebSOCTimeTo24HourTime = (time: string) => {
```

Comment:
Can anybody confirm if changing this interface breaks any part of the app? 

The rationale for the API change is ***don't pass in the entire object  to the function if all it needs is a string*** to do its work